nginx/nginx.conf

```diff
@@ -0,0 +1,31 @@
+#user       www www;  ## Default: nobody
+#worker_processes  5;  ## Default: 1
+#error_log  logs/error.log;
+#pid        logs/nginx.pid;
+#worker_rlimit_nofile 8192;
+
+events {
+}
+
+http {
+    server {
+        listen 8080 default_server;
```

Comment:
I would typically just use port 80 in this to be more standard, is there a reason to prefer 8080 on the guest?

Reply:
sure, used 8080 for development / testing as I had something else running at 80. will Change it to 80 for final version